src/container/rpc.h

```diff
@@ -370,6 +373,16 @@ CRT_RPC_DECLARE(cont_tgt_snapshot_notify, DAOS_ISEQ_CONT_TGT_SNAPSHOT_NOTIFY,
 
 CRT_RPC_DECLARE(cont_prop_set, DAOS_ISEQ_CONT_PROP_SET, DAOS_OSEQ_CONT_PROP_SET)
 
+#define DAOS_ISEQ_CONT_ACL_UPDATE	/* input fields */	 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
False positive